arm9/lib/src/MI_init.c

```diff
@@ -0,0 +1,10 @@
+#include "nitro/types.h"
+#include "function_target.h"
+#include "MI_dma.h"
+#include "MI_wram.h"
+
+ARM_FUNC void MI_Init(void)
+{
+    MI_SetWramBank(MI_WRAM_ARM7_ALL);
```

Comment:
Nani?!

Reply:
not sure what's wrong with this? MI_WRAM_ARM7_ALL is the last value in the MIWram enum

Reply:
I'm just surprised to see ARM7 in the ARM9 source, is all.

Reply:
yeah it is kinda odd, but that's what it is according to the sdk /shrug